dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -105,6 +105,8 @@ static String getCleanedDescription(String description) {
             if (description.length() > MAX_DESCRIPTION) {
                 description = description.substring(0, MAX_DESCRIPTION - Client.PADDING) + "...";
             }
+        } else {
```

Comment:
Nitpick: 
Can probably remove both the if and the else (saving three lines) by starting with `description = MoreObjects.firstNonNull(description, "")` right at the top.